scripts/bundleSize.sh

```diff
@@ -2,8 +2,8 @@
 
 # Size limit for all bundles used by each service (K)
 # Keep these +/- 5K and update frequently!
-min=557
-max=576
+min=560
+max=580
```

Comment:
The 3 functions only push the news JS bundle size from 576 to 577 - just adding some buffer

Reply:
The buffer should be 5K from the biggest service and 5k from smallest

Reply:
Doesn't look like people are following that rule then :(  Updated the thresholds